events-and-workshops.md

```diff
@@ -1,4 +1,14 @@
-# 📅  Events & Workshops
+# <span class="emoji-bullet">📅</span> Events & Workshops
+
+## Calendar
+
+This calendar has dates for upcoming events in the lab.
+To subscribe directly, the [calendar URL is here](https://calendar.google.com/calendar/embed?src=h4tfirrturtt83oamhht396uv8%40group.calendar.google.com&ctz=America%2FToronto)
+
+<iframe src="https://calendar.google.com/calendar/embed?src=h4tfirrturtt83oamhht396uv8%40group.calendar.google.com&ctz=America%2FToronto" style="border: 0" width="100%" height="600" frameborder="0" scrolling="no"></iframe>
```

Comment:
Should we be making this public?

Right now, all of our events contain a zoom link with an embedded password. So, theoretically anyone lurking the website could see this page and hop into one of our meetings.

Reply:
ah, this is a very good point.

my philosophy is "better sorry than safe". I.e., if one day we get zoom-boomed we will change our practice, but if we don't and everything is going smoothly, then there is no need to take extra security layers.

the pros in making this public is that lab members and collaborators have this info more handy

Reply:
I had the same concern as @joshuacwnewton. We're pretty exposed to zoombombing and vandalism in our meeting minutes in our current posture.

Reply:
we keep using the same zoom link for the past 3 years and yet we've never been zoom-bombed 🤞